python/cugraph/cugraph/structure/graph_implementation/simpleGraph.py

```diff
@@ -106,6 +106,17 @@ def __from_edgelist(
                 "names not found in input. Recheck the source and "
                 "destination parameters"
             )
+        df_columns = s_col + d_col
+
+        # Drop the edge column(s) if edge_attr=None
+        if edge_attr is None:
+            input_df = input_df[df_columns]
+        else:
+            if not (set([edge_attr]).issubset(set(input_df.columns))):
```

Comment:
I've seen this pattern before but I'm wondering if it's worth the readability for a list of size 1. It seems like this could be about as fast for this and is more readable
```
if edge_attr not in input_df.columns:
   ...
```

Reply:
And do I also add an `elif` in case `edge_attr` happens to be a `list`? for instance It is common for multicolumn vertices to have a list of column names and I believe this `set().isubset()` was meant to encompass both cases (list and str).

Reply:
That's a good point, I'll just drop this and we'll leave this code as-is.